style.css

```diff
@@ -0,0 +1,303 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@700&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Montserrat', sans-serif;
+}
+
+/* Nabvar */
+
+.navbar {
+    position: absolute;
+    top: 0;
+    left: 0;
+    display: flex;
+    width: 100%;
+    align-items: center;
+    justify-content: space-between;
+    padding-top: 20px;
+    padding-left: 11rem;
+    padding-right: 11rem;
+}
+
+.navbar__log {
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 32px;
+    letter-spacing: 0.1px;
+    color: #252B42;
+} 
+
+.navbar__logo-links {
+    display: flex;
+    gap: 5rem;
+}
+
+.navbar__links {
+    display: flex;
+    align-items: center;
+    list-style: none;
+    gap: 3rem;
+}
+
+
+.link {
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 24px;
+    letter-spacing: 0.2px;
+    color: #737373;
+
+}
+
+.link__text {
+    line-height: 22px;
+    color: #1BA0AB;
+    text-decoration: none;
+}
+
+.btn {
+    padding: 15px 25px;
+    gap: 15px;
+    width: 137px;
+    height: 52px;
+    background: #1BA0AB;
+    border-radius: 5px;
+    color: white;
+    text-decoration: none;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 22px;
+    letter-spacing: 0.2px;
+    text-align: center;
+}
+
+/* Header */
+
+header {
```

Comment:
## style wrong
this is not bem find a name for this tag

Reply:
use classes to style not tags